### DISCUSSION TITLE
Using custom objectId's

### DISCUSSION THREAD

**Discussion by vdkdamian** [netreconlab/Parse-Swift#18] [Q&A] [answered]
Hello, I enabled custom objectId's with `requiringCustomObjectIds: true`. I am trying to get this working, but i'm getting an error:
`ParseError code=105 error=objectId is an invalid field name.`

Anyone who knows how to fix this?

**Comment by cbaker6** [comment, marked as answer]
When are you getting this error? Is it during a save? Also, can you post your code? My guess is you don’t have `allowCustomObjectId` enabled on your server: https://github.com/netreconlab/parse-hipaa/blob/04792bdf271fcd506788ef4987f4393a166941d8/parse/index.js#L181-L195

which gets passed to: https://github.com/netreconlab/parse-hipaa/blob/04792bdf271fcd506788ef4987f4393a166941d8/parse/index.js#L292

**Reply by vdkdamian** [reply]
```
    let list = PGRList(name: "TESTT")
    Task {
        do {
            let result = try await list.save()
            print(result)
        } catch let errorr {
            print("errorr: \(errorr)")
        }
    }
```

I indeed did not enable it on the server, but I have no idea how to do this

**Reply by cbaker6** [reply]
It’s in your server config file. If you use a hosting service, you should ask them to enable it.

**Reply by vdkdamian** [reply]
I found how to enable it

**Reply by vdkdamian** [reply]
Thanks!
